src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": ">=5.5.9"
     },
     "require-dev": {
-        "symfony/yaml": "~2.8|~3.0",
+        "symfony/yaml": "~3.2",
```

Comment:
this line does apply only when testing the component, it has no effect when used as a dep of something else